perfdash/config.go

```diff
@@ -623,13 +623,24 @@ var (
 		},
 	}
 
+	watchListDescriptions = TestDescriptions{
+		"E2E": {
+			"LoadResources": []TestDescription{{
+				Name:             "load",
```

Comment:
This should be `watch-list` I think? Based on file in artifacts ` ResourceUsageSummary_watch-list_2023-05-24T08:35:20Z.json`

Reply:
I would also add something that would allow me to show latencies

Reply:
perhaps
```
"LoadResponsiveness": []TestDescription{{
				Name:             "watch-list",
				OutputFilePrefix: "APIResponsiveness",
				Parser:           parsePerfData,
			}},
```

?

Reply:
or

```
"LoadResponsiveness_PrometheusSimple": []TestDescription{{
				Name:             "watch-list",
				OutputFilePrefix: "APIResponsivenessPrometheus_simple",
				Parser:           parsePerfData,
			}},
```
?

Reply:
If you want to you can run perfdash locally to play with and find all the metrics that you need. To do that modify perfdash/Makefile by replacing githubConfigDir with  `https://api.github.com/repos/kubernetes/test-infra/contents/config/jobs/kubernetes/sig-scalability` and replace builds with some reasonable number like 2 so you don't have to wait minutes for perdash to scrape all the data. After that just run `make run` from perfdash directory and you should have perfdash hosted at localhost:8080